### PR TITLE
[JESCorrections] small updates to `fitJESCs` (`Run3Winter20_V2_MC`)

### PR DESCRIPTION
**Pull request by missirol**
Small follow-ups to #43, including more compact info in readme.

@sparedes, please double-check that the changes are correct (i.e. do not introduce changes in outputs).

FYI: @pallabidas

### CODE REVIEW

**Comment by missirol** [comment]
@sparedes, could you have a look and sign off on these changes? thanks

File: JESCorrections/test/fitJESCs [hunk 1]
```diff
@@ -175,9 +170,12 @@ runJECWorflowForOneAlgo(){
   if [ ! -f ${stepName}.done ]; then
     CMD_STEP="jet_l2_correction_x -algs ${jetType} -era ${ERA} -l2l3 true"
     CMD_STEP+=" -input plots_${stepNameOld}/histogram_${jetType}l1_${stepNameOld}.root -outputDir ./ -output l2p3.root"
-    CMD_STEP+=" -makeCanvasVariable AbsCorVsJetPt:JetEta -batch true -histMet median -l2pffit standard+gaussian -maxFitIter 50 -ptclipfit true" # for all other jet collections
-    # for ak8puppi only
-    #CMD_STEP+=" -makeCanvasVariable AbsCorVsJetPt:JetEta -batch true -histMet median -l2pffit standard+Gaussian -maxFitIter 50 -ptclipfit false" # for ak8puppi only
+    CMD_STEP+=" -makeCanvasVariable AbsCorVsJetPt:JetEta -batch true -histMet median -l2pffit standard+gaussian -maxFitIter 50 -ptclipfit false"
+    if [ "X${jetType}" != "Xak8puppi" ]; then
```
Comment (sparedes) [inline JESCorrections/test/fitJESCs]:
```suggestion
    if [ "X${jetType}" != "Xak8puppiHLT" ]; then
```

**Comment by sparedes** [comment]
@missirol  only one small change, otherwise it looks good!

**Comment by missirol** [comment]
thanks @sparedes

**Pull request merged**